docs/treat to vanilla-extract migration.md

```diff
@@ -3,6 +3,8 @@
 This document provides guidance for consumers looking to migrate Braid-based styles from [treat](https://seek-oss.github.io/treat) to [vanilla-extract.](http://vanilla-extract.style)
 
 Since they use different file extensions (`.css.ts` vs `.treat.ts`), we're able to ease the migration by supporting both approaches simultaneously in the same project.
+However, Braid v31 was the last version to support treat styles in addition to vanilla-extract styles.
+From v32 onwards, treat styles are no longer supported and should be migrated to vanilla-extract.
```

Comment:
This reads a bit weird, even in the context of the paragraph. Will re-write this.